CHANGELOG.md

```diff
@@ -26,7 +26,11 @@
 
 - Added support for `Index.to_numpy`.
 - Added support for `DataFrame.align` and `Series.align` for `axis=0`.
-- Added support for `size` in `GroupBy.aggregate`, `DataFrame.aggregate`, and `Series.aggregate`.
+- Added support for aggregation functions `"size"` and `len` in `GroupBy.aggregate`, `DataFrame.aggregate`, and `Series.aggregate`.
```

Comment:
nit: Is there a reason `size` is in quotes but len isn't?

Reply:
You can agg with the function `len` but not the string `"len"`, and you can agg with the string `"size"` but usually not with `size` :)

Reply:
Are these limitations coming from native pandas?!

Reply:
`size` is a variable that's not bound to anything in python. the note here was wrong before my change since we could aggregate by `"size"` but not by `size`.